note/doc/index.md

```diff
@@ -0,0 +1,83 @@
+```js script
+import { html } from 'lit';
+import '@divriots/dockit-core/note/dockit-note.define';
+```
+
+# dockit-note
+
+Component used to call out information in documentation pages.
+
+## Setup
+
+```js
+import '@divriots/dockit-core/note/dockit-note';
+```
+
+## Demo
+
+```js preview-story
+export const note = () => html`
+  <dockit-note>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit
+  </dockit-note>
+`;
+```
+
+### Variants
+
+#### Info
+
+```js preview-story
+export const note_info_variant = () => html`
+  <dockit-note variant="info">
```

Comment:
I have same feeling here as with https://github.com/divriots/dockit-core/pull/75/files#r852808027

users should be able to do

```css
dockit-note {
  --dockit-note-color: #123123;
}
dockit-note[variant=info] {
  --dockit-note-color: #456456;
}
dockit-note[variant=warning] {
  --dockit-note-color: #789789;
}
/* and so on */
```

Reply:
The  color attribute here is still nice if the user wants to have a single custom color once. I think it's OK for a `note`.

But I definitely like the idea of 
```
dockit-note { --dockit-note-color: #123123; } 
dockit-note[variant=info] { --dockit-note-color: #456456; } 
dockit-note[variant=warning] { --dockit-note-color: #789789; }
```

because users can define all the variants they want and apply the color style centrally in CSS. Separates the semantic from the style 👍